src/ci/docker/host-x86_64/dist-x86_64-linux/build-gccjit.sh

```diff
@@ -20,9 +20,14 @@ cd ../gcc-build
     --enable-checking=release \
     --disable-bootstrap \
     --disable-multilib \
-    --prefix=$(pwd)/../gcc-install
-make
-make install
+    --prefix=$(pwd)/../gcc-install \
+    --quiet
+
+# We redirect stdout because the GCC build scripts generate thousands of lines
+# of output. Unfortunately, even when using `--quiet` option for all commands,
+# the output still thousands of lines of output, forcing us to use this solution.
+make > /dev/null
+make install > /dev/null
```

Comment:
Please use hide_output from src/ci/docker/scripts/shared.sh (feel free to grep for other usages). That makes sure there's some progress indication and we do the right thing on failure.